### PR TITLE
Create release draft on new tags with desktop applications and apk attached

### PR DESCRIPTION
**Pull request by perclasson**
Simplifies the release process to automatically create release drafts with desktop applications and apk included:

- Create release draft when a tag is added
- Update os to use fixed version instead of latest (a Google best practice)

Relates to https://github.com/flutter/gallery/issues/43

Was tested on my personal repo https://github.com/perclasson/gallery/actions/runs/261192033, see finished example release:
https://github.com/perclasson/gallery/releases/tag/v2.8

### CODE REVIEW

**Comment by guidezpl** [comment]
So neat!

**Review by rami-a — APPROVED** [review]
LGTM

**Review by guidezpl — APPROVED** [review]
```
  _    ___ _____ __  __ 
 | |  / __|_   _|  \/  |
 | |_| (_ | | | | |\/| |
 |____\___| |_| |_|  |_|
                        
```

**Review by clocksmith — APPROVED** [review]

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+        flutter create .
+        flutter build -v ${{ matrix.target }} --release
+      working-directory: code
+    - name: Compress build for Linux and macOS
+      if: startsWith(matrix.os, 'macOS') || startsWith(matrix.os, 'ubuntu')
+      run: zip -r $GITHUB_WORKSPACE/flutter_gallery_${{ matrix.target }} .
+      working-directory: code/${{ matrix.path }}
+    - name: Compress build for Windows
+      if: startsWith(matrix.os, 'windows')
+      run: compress-archive -Path * -DestinationPath ${env:GITHUB_WORKSPACE}\flutter_gallery_${{ matrix.target }}.zip
```
Comment (stuartmorgan) [inline .github/workflows/release.yml:100]: Not that this won't run for all users; see https://github.com/flutter/flutter/wiki/Desktop-shells#distributing

If a zip is the distribution method the C++ redistributables should be included in it (i.e., copied into this folder before making the archive).

Comment (perclasson) [reply .github/workflows/release.yml:100]: Makes sense!

@stuartmorgan I'm adding them here: https://github.com/flutter/gallery/pull/320

**Pull request merged**